content/docs/installation/supported-releases.md

```diff
@@ -22,9 +22,9 @@ and other world events.
 
 ## Upcoming releases
 
-| Release  |      Release Date      |      End of Life      | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
-|----------|:----------------------:|:---------------------:|:----------------------------------:|:---------------------------------:|
-| [1.13][] | End of September, 2024 | End of November, 2024 |            1.22 → 1.27             |            4.9 → 4.14             |
+| Release  |      Release Date      |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
+|----------|:----------------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
+| [1.13][] | End of July, 2024      | End of September, 2024 |            1.22 → 1.27             |            4.9 → 4.14             |
```

Comment:
Hi, @maelvls, 

I've found that the release date of 1.13 is scheduled as `2024` in this page. However I think it should be 2023.

```suggestion
| [1.13][] | End of July, 2023      | End of September, 2023 |            1.22 → 1.27             |            4.9 → 4.14             |
```

Also, the release date and EOL date of 1.12 should be 2023. Am I wrong?

Reply:
Hey, thank you @rinx!